Game/Source/GDKTestGyms/Private/RPCTimeoutPC.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "RPCTimeoutPC.h"
+
+
+#include "Chaos/PhysicalMaterials.h"
+#include "Components/TextRenderComponent.h"
+#include "Engine/StreamableManager.h"
+#include "GameFramework/Character.h"
+#include "UObject/ConstructorHelpers.h"
+
+ARPCTimeoutPC::ARPCTimeoutPC()
+{
+	//Choose materials which belong to the Engine. This is in anticipation of the possibility of moving this test to the UnrealGDK plugin in the future.
+	
+	static ConstructorHelpers::FObjectFinder<UMaterial>FailedMaterialFinder(TEXT("Material'/Engine/EngineDebugMaterials/VertexColorViewMode_RedOnly.VertexColorViewMode_RedOnly'"));
+	FailedMaterialAsset = FailedMaterialFinder.Object;
+	
+	SoftMaterialPath = FSoftObjectPath(TEXT("Material'/Engine/Tutorial/SubEditors/TutorialAssets/Character/TutorialTPP_Mat.TutorialTPP_Mat'"));
+	SoftMaterialPtr = TSoftObjectPtr<UMaterial>(SoftMaterialPath);
+}
+
+void ARPCTimeoutPC::Tick(float DeltaTime)
+{
+	Super::Tick(DeltaTime);
```

Comment:
This override doesn't seem to be doing anything?

Reply:
Missed it, you're right its unnecessary